target/linux/realtek/image/Makefile

```diff
@@ -19,6 +19,15 @@ define Device/Default
 	append-metadata | check-size
 endef
 
+define Build/custom-uimage
+	mkimage -A $(LINUX_KARCH) \
+		-O linux -T kernel \
+		-C gzip -a $(KERNEL_LOADADDR) $(if $(UIMAGE_MAGIC),-M $(UIMAGE_MAGIC),) \
+		-e $(if $(KERNEL_ENTRY),$(KERNEL_ENTRY),$(KERNEL_LOADADDR)) \
+		-n '$(1)' -d $@ $@.new
+	mv $@.new $@
+endef
```

Comment:
This has been dropped in https://github.com/openwrt/openwrt/commit/8582e15616d3d42340810cb9c880825ccf87e1b0

Please use shared Build/uImage instead.